interface/therapy_groups/therapy_groups_models/users_model.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * interface/therapy_groups/therapy_groups_models/users_model.php contains the model for users.
+ *
+ * This model fetches the users data to be used for the therapy group from the DB.
+ *
+ * Copyright (C) 2016 Shachar Zilbershlag <shaharzi@matrix.co.il>
+ * Copyright (C) 2016 Amiel Elboim <amielel@matrix.co.il>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  Shachar Zilbershlag <shaharzi@matrix.co.il>
+ * @author  Amiel Elboim <amielel@matrix.co.il>
+ * @link    http://www.open-emr.org
+ */
+
+class Users{
```

Comment:
In concept it's a good idea but it causes problems with our loadModel in base_controller.php:
```
    protected function loadModel($name)
    {
        if(!isset($this->$name)){
            require(dirname(__FILE__) .'/../' . self::MODEL_FOLDER .'/'. strtolower($name) .'_model.php');
            $this->$name = new $name;
        }

        return $this->$name;
    }
```
This method loads both the class and the file using the namespace, so if we pass on the namespace as well, it will cause a problem here.
I'm not sure we can change this right now.